tigerpath/majors_and_certificates/majors/ECO_2018.json

```diff
@@ -25,227 +25,261 @@
+      "req_list": [
+        {
+          "name": "Electives",
+          "max_counted": null,
+          "min_needed": 3,
+          "explanation": null,
+          "course_list": [
+            "ECO 3*",
+            "ECO 4*",
+            "ECO 5*"
+          ]
+        },
+        {
+          "name": "Cognates",
+          "max_counted": 2,
+          "min_needed": null,
+          "explanation": "Economics majors are allowed to count a maximum of two courses from other departments as cognates. These courses need to have substantial economics content. A course with an Economics cross-listing (indicated by an ECO 3XX, ECO 4XX, or ECO 5XX number in its first or second listing) counts as a regular departmental, not as a cognate. Courses that will be automatically recognized by SCORE as cognates are listed below. No application is necessary for these courses.",
+          "course_list": [
+            "COS 445",
+            "ELE 381",
+            "HIS 474",
+            "MAT 378",
+            "ORF 307",
+            "ORF 309",
+            "ORF 311",
+            "ORF 350",
+            "ORF 360",
+            "ORF 363",
+            "ORF 405",
+            "ORF 417",
+            "ORF 474",
+            "POL 335",
+            "POL 347",
+            "POL 349",
+            "POL 352",
+            "POL 385",
+            "WWS 309",
+            "WWS 340/PSY 321",
+            "WWS 373/CHV 373",
+            "WWS 408",
+            "WWS 466/HIS 467",
+            "WWS 524",
+            "WWS 538",
+            "WWS 582C",
+            "WWS 582F"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Tracks",
       "max_counted": 1,
       "min_needed": 1,
-      "description": "Standard track or Politcal Economy",
-      "explanation": "The Department offers a Program in Political Economy for Economics concentrators interested in studying issues at the intersection of these two fields. Program participants meet all the general requirements of Economics concentrators, and, in addition: (1) complete two Politics courses at any level before entering the program; (2) complete three 300-level (or higher) Politics courses during junior and senior years, and (3) conduct junior independent work with a political economy component. Politics courses approved as Economics cognates may also count toward the program’s requirements. Admission to the program takes place at the beginning of the junior year, in consultation with the Political Economy Adviser in Economics. A student who participates in this program is an Economics concentrator and is subject to all the requirements of the Economics department.  Concentrators who successfully complete the program’s requirements will receive a departmental attestation.",
+      "explanation": null,
       "req_list": [
         {
-          "name": "Economics (Standard)",
+          "name": "Economics",

```

Comment:
If adopting the hack (see comment on line 213), you can then remove this redundant track.